test/integration/conftest.py

```diff
@@ -402,3 +410,33 @@ def b2_uri_args(apiver_int):
         return b2_uri_args_v4
     else:
         return b2_uri_args_v3
+
+
+# -- Persistent bucket fixtures --
+@pytest.fixture
+def persistent_bucket(b2_api) -> Bucket:
+    return get_or_create_persistent_bucket(b2_api)
+
+
+@pytest.fixture
+def unique_subfolder():
+    subfolder = f"test-{uuid.uuid4().hex[:8]}"
+    yield subfolder
+
+
+@pytest.fixture
+def persistent_bucket_aggregate(
```

Comment:
nitpick:
shorter name would be easier
```suggestion
def persistent_bucket(
```

btw, this could have been B2 URI object, but again, doesn't matter much